internal/encryption/keyprovider/aws_kms/config.go

```diff
@@ -92,6 +92,10 @@ func (c Config) asAWSBase() (*awsbase.Config, error) {
 	if err != nil {
 		return nil, err
 	}
+	var roles []awsbase.AssumeRole
```

Comment:
`note`
Upgraded aws/s3 lib => rendered some tests in `backend_complete_test.go` to fail => forced upgrading hashicorp/aws-sdk-go-base/v2 => forced upgrading the way assumed role(s) are passed over.